backend/src/test/java/com/staccato/fixture/category/CategoryFixtures.java

```diff
@@ -57,24 +58,31 @@ public CategoryBuilder withColor(Color color) {
             return this;
         }
 
-        public Category build() {
-            return new Category(thumbnailUrl, title, description, color, term.getStartAt(), term.getEndAt());
+        public CategoryBuilder withIsShared(Boolean isShared) {
+            this.isShared = isShared;
+            return this;
         }
 
-        public Category buildWithMember(Member member) {
-            Category category = build();
-            category.addCategoryMember(member);
+        public CategoryBuilder withHost(Member member) {
+            this.hosts.add(member);
+            return this;
+        }
+
+        public CategoryBuilder withGuest(Member member) {
```

Comment:
```suggestion
        public CategoryBuilder withGuest(Member... members) {
```
와 같은 방식으로 가변 인자를 활용해보면 어떨까요?
이 구조를 사용한다면, 여러 명의 Guest를 추가하고 싶을 때 `withGuest`를 중첩해서 사용할 필요없이, 한 번의 `withGuest`로 1명 이상의 Guest 추가가 가능할 것 같아요!

Reply:
제 피드백 적용해주셔서 감사합니다 ㅎㅎ
가변 인자 얘기해주셨던 내용인데 다른거 반영하면서 그냥 넘겨버렸나봐요 ㅠㅠ

가변 인자를 사용할지, 리스트를 사용할지 고민해봤는데, 리스트를 사용하면 한 명의 게스트를 넘길 때에도 리스트로 감싸야 해서 가독성이 떨어진다는 단점이 있네요. 실제로도 한 명의 게스트를 넘기는 경우가 많을 것 같아, 가변 인자가 좋을 것 같습니다!

```java
public CategoryBuilder withGuests(Member... members) {
    this.guests.addAll(Arrays.asList(members));
    return this;
}
```